### PR TITLE
Doodlise 

### PR DESCRIPTION
**Pull request by sushinoya**

### CODE REVIEW

**Comment by 24r** [comment]
Deploy preview for *showface* ready!

Built with commit 9e3a77115b0714aed4f4d92bccfecfdd49c4a649

https://deploy-preview-18--showface.netlify.com

**Comment by taneliang** [comment]
@sushinoya The new design looks pretty good! I think we can try vertically centering the entire blue box onscreen, and the orange bottom bar shouldn't be wider than the blue box.

Also, remember to run Prettier (`yarn prettify`) before merging. Your coding style and indentation are quite different from everything else in this repo.

File: src/components/CreatePage.js [hunk 1]
```diff
@@ -118,9 +118,16 @@ class CreatePage extends Component {
...
+           <div>{this.renderContent()}</div>
         </section>
-      </div>
+        {/* <div className={styles.appBarBottom}></div> */}
+        <BottomAppBar 
+          content={<Button 
+            className={styles.submitButton} 
+            onClick={this.handleSubmit}
+            raised>Submit</Button>} 
+          />
```
Comment (taneliang) [inline src/components/CreatePage.js]: Any reason why this `content` prop isn't called `children` instead? That'll allow you to do something like the suggested change. It's not formatted correctly but if you run Prettier it'll auto format for us.

```suggestion
        <BottomAppBar>
          <Button 
            className={styles.submitButton} 
            onClick={this.handleSubmit}
            raised>Submit</Button>
        </BottomAppBar>
```

Comment (sushinoya) [reply src/components/CreatePage.js]: yeah next commit i made it props.children alr

**Pull request merged**